tests/scalers/azure-data-explorer.test.ts

```diff
@@ -0,0 +1,237 @@
+import * as fs from 'fs'
+import * as sh from 'shelljs'
+import * as tmp from 'tmp'
+import test from 'ava'
+
+const dataExplorerDb = process.env['TEST_DATA_EXPLORER_DB']
+const dataExplorerEndpoint = process.env['TEST_DATA_EXPLORER_ENDPOINT']
```

Comment:
Could you please add `AZURE_` prefix to these envs?

Reply:
`AZURE_DATA_EXPLORER_TEST_DB`
`AZURE_DATA_EXPLORER_TEST_ENDPOINT` 

maybe to something like this?

Reply:
That's what  I'm adding in https://github.com/kedacore/keda/pull/2643 indeed

Reply:
So these envs were added as:
```
       AZURE_DATA_EXPLORER_DB: ${{ secrets.AZURE_DATA_EXPLORER_DB }}
       AZURE_DATA_EXPLORER_ENDPOINT: ${{ secrets.AZURE_DATA_EXPLORER_ENDPOINT }}
```

Reply:
done